torchaudio/transforms.py

```diff
@@ -283,6 +283,15 @@ def __init__(self,
 
         assert f_min <= self.f_max, 'Require f_min: {} < f_max: {}'.format(f_min, self.f_max)
 
+        if n_stft is None or n_stft == 0:
+            warnings.warn(
+                'Initialization of torchaudio.transforms.MelScale with an unset weight '
+                '`n_stft` is deprecated and will be removed from a future release. '
+                'Please set a proper n_stftvalue. Typically this isn_fft // 2 + 1. '
```

Comment:
```suggestion
                'Please set a proper `n_stft` value. Typically this is `n_fft // 2 + 1`. '
```
Looks like spaces are lost here.